awswrangler/neptune/_neptune.py

```diff
@@ -110,8 +110,8 @@ def execute_sparql(client: NeptuneClient, query: str) -> pd.DataFrame:
     data = client.read_sparql(query)
     df = None
     if "results" in data and "bindings" in data["results"]:
-        df = pd.DataFrame(data["results"]["bindings"])
-        df.applymap(lambda x: x["value"])
+        df = pd.DataFrame(data["results"]["bindings"], columns=data.get("head").get("vars"))
+        df = df.applymap(lambda d: d["value"] if "value" in d else None)
     else:
         df = pd.DataFrame(data)
```

Comment:
Does it make sense to add `columns=data.get("head", {}).get("vars")` here as well?